source/utils/types.ts

```diff
@@ -59,10 +73,16 @@ export interface Options extends RequestOptions {
 	decompress?: boolean;
 	encoding?: BufferEncoding | null;
 	method?: Method;
+	retry?: RetryDescriptor;
+	throwHttpErrors?: boolean;
 	// TODO: Remove this once TS migration is complete and all options are defined.
 	[key: string]: unknown;
 }
 
+export interface RequestEmitter extends EventEmitter {
```

Comment:
Isn't there a built-in type we can use for this?

Reply:
You're right, there is. See https://github.com/sindresorhus/got/pull/720/commits/775684bb9fe153bbf641d8a90993cede216f7c0e for the change.